### DISCUSSION TITLE
Multi Database support

### DISCUSSION THREAD

**Discussion by ishaan812** [prest/prest#900] [General] [answered]
Hey folks,
Was looking for a multi database solution.
The documentation only mentioned:
Since [v1.1.2](https://github.com/prest/prest/releases/tag/v1.1.2) it is a lot safer to use multiple databases, but not yet in the ideal state of security that we want, so use it in your own risk.

If someone could help me out on how to configure this using the toml. Can I support something like have two DB urls and based on the database name, dynamically change the dbconnection for the request? Wanted to just figure out if this is already supported by default.

**Comment by arxdsilva** [comment, marked as answer]
Hi @ishaan812 please email me and we can chat longer about your situation: arthur@prestdlabs.com

yes you can have two DB urls and change the connection dynamically.
